fedoratagger/lib/retired.py

```diff
@@ -3,69 +3,71 @@
 Is that the script Should be run as:
 
 FEDORATAGGER_CONFIG = /etc/fedora-tagger/fedora-tagger.cfg python /path/to/fedoratagger/lib/retired.py
-
-or
-
-FEDORATAGGER_CONFIG = /etc/fedora-tagger/fedora-tagger.cfg fedoratagger-remove-pkgs -s Retired
 """
 import requests
-import json
 
-import argparse
 import model as m
 import fedoratagger as ft
 
-import sqlalchemy
-
 import logging
 
 log = logging.getLogger("fedoratagger-remove")
 log.setLevel(logging.DEBUG)
 logging.basicConfig()
 
-PKGDB_URL = 'https://admin.fedoraproject.org/pkgdb/api/packages'
+PDC_URL = 'https://pdc.fedoraproject.org/rest_api/v1/component-branches'
+
+
+def get_retired_packages():
 
-def get_packages(status):
-    log.info('Retrieve the list of %s packages.' % status)
-    total = 1
-    page = 1
     packages = []
-    while page <= total:
-        args = {
-            'status': status,
-            'page': page,
-        }
-        r = requests.get(PKGDB_URL, params=args)
-        output = json.loads(r.text)
-        if hasattr(output, 'error'):
-            log.error(output['error'])
-            return packages
-
-        total = output['page_total']
-        for pkg in output['packages']:
-            packages.append(pkg['name'])
-        page += 1
+
+    args = {
+        'name': 'f28',
+        'active': False,
+    }
+
+    r = requests.get(PDC_URL, args)
+    output = r.json()
```

Comment:
it would be better to test the response status before calling `.json()` method. If the response status is not 200 then this will fail. 
You can look at the requests response `status_code` (http://docs.python-requests.org/en/master/api/#requests.Response).